sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDatabase.java

```diff
@@ -467,8 +506,8 @@ public Integer readProvisionedThroughput() {
     /**
      * Replace provisioned throughput integer.
      *
-     * @param requestUnitsPerSecond the request units per second
-     * @return the integer
+     * @param requestUnitsPerSecond the request units per second.
+     * @return the integer.
      */
     public Integer replaceProvisionedThroughput(int requestUnitsPerSecond) {
```

Comment:
Same here : hide / remove this API - https://github.com/Azure/azure-sdk-for-java/issues/11191

Reply:
Fixed.